### PR TITLE
SSVD-2613 UX for Hue when adding lights is confusing

### PR DESCRIPTION
**Pull request by larsfinander**
-Split lights list in SmartApp into already added lights and new lights

Problem was that user would expect devices to be removed if you uncheck them in the list of light but they arent.

### CODE REVIEW

**Comment by larsfinander** [comment]
@juano2310 @workingmonk Does one of you have time for another VD code review?

File: smartapps/smartthings/hue-connect.src/hue-connect.groovy [hunk 1]
```diff
@@ -133,29 +133,49 @@ def bulbDiscovery() {
...
+				 existingLightsDescription += ", ${it.value}"
+			}
+		}
+	}
 
 	return dynamicPage(name:"bulbDiscovery", title:"Light Discovery Started!", nextPage:"", refreshInterval:refreshInterval, install:true, uninstall: true) {
 		section("Please wait while we discover your Hue Lights. Discovery can take five minutes or more, so sit back and relax! Select your device below once discovered.") {
-			input "selectedBulbs", "enum", required:false, title:"Select Hue Lights (${numFound} found)", multiple:true, options:bulboptions
+			input "selectedBulbs", "enum", required:false, title:"Select Hue Lights to add (${numFound} found)", multiple:true, options:newLights
+			paragraph title: "Previously added Hue Lights (${existingLights.size()} added)", existingLightsDescription
```
Comment (workingmonk) [inline smartapps/smartthings/hue-connect.src/hue-connect.groovy:175]: did you want the comma between the title and the object?

Comment (larsfinander) [reply smartapps/smartthings/hue-connect.src/hue-connect.groovy:175]: Yes, it does not work properly without it!

Comment (workingmonk) [reply smartapps/smartthings/hue-connect.src/hue-connect.groovy:175]: `paragraph title: "Previously added Hue Lights (${existingLights.size()} added) $existingLightsDescription"`
^^ this won't work?

Comment (larsfinander) [reply smartapps/smartthings/hue-connect.src/hue-connect.groovy:175]: If done like it is now, the description is black and listed devices are in grey which i think is good

**Comment by workingmonk** [comment]
👍

**Pull request merged**